css/style.css

```diff
@@ -1 +1 @@
idth:4px}.buildings-cloud_three{height:16px;position:relative;width:24px}.buildings-cloud_three:after{box-shadow:4px 4px transparent,8px 4px transparent,12px 4px #24346e,16px 4px #24346e,20px 4px transparent,24px 4px transparent,4px 8px transparent,8px 8px #24346e,12px 8px #24346e,16px 8px #24346e,20px 8px #24346e,24px 8px transparent,4px 12px #24346e,8px 12px #24346e,12px 12px #24346e,16px 12px #24346e,20px 12px #24346e,24px 12px #24346e,4px 16px transparent,8px 16px #24346e,12px 16px #24346e,16px 16px #24346e,20px 16px #24346e,24px 16px transparent;content:"";height:4px;left:-4px;position:absolute;top:-4px;width:4px}.buildings-cloud_four{height:20px;position:relative;width:48px}.buildings-cloud_four:after{box-shadow:4px 4px transparent,8px 4px transparent,12px 4px #24346e,16px 4px #24346e,20px 4px #24346e,24px 4px transparent,28px 4px transparent,32px 4px #24346e,36px 4px #24346e,40px 4px transparent,44px 4px transparent,48px 4px transparent,4px 8px transparent,8px 8px #24346e,12px 8px #24346e,16px 8px #24346e,20px 8px #24346e,24px 8px #24346e,28px 8px #24346e,32px 8px #24346e,36px 8px #24346e,40px 8px #24346e,44px 8px #24346e,48px 8px transparent,4px 12px transparent,8px 12px #24346e,12px 12px #24346e,16px 12px #24346e,20px 12px #24346e,24px 12px #24346e,28px 12px transparent,32px 12px #24346e,36px 12px #24346e,40px 12px transparent,44px 12px #24346e,48px 12px #24346e,4px 16px transparent,8px 16px transparent,12px 16px #24346e,16px 16px #24346e,20px 16px #24346e,24px 16px #24346e,28px 16px #24346e,32px 16px #24346e,36px 16px #24346e,40px 16px #24346e,44px 16px #24346e,48px 16px transparent,4px 20px transparent,8px 20px transparent,12px 20px transparent,16px 20px transparent,20px 20px #24346e,24px 20px #24346e,28px 20px #24346e,32px 20px #24346e,36px 20px #24346e,40px 20px transparent,44px 20px transparent,48px 20px transparent;content:"";height:4px;left:-4px;position:absolute;top:-4px;width:4px}.modal{-webkit-animation-name:fadeIn;animation-name:fadeIn;background-color:#000;background-color:rgba(0,0,0,.6);display:none;height:100%;left:0;overflow:auto;z-index:1000}.modal,.modal-content{-webkit-animation-duration:.4s;animation-duration:.4s;position:fixed;top:0;width:100%}.modal-content{-webkit-animation-name:slideIn;animation-name:slideIn;background-color:#0c1021;border-bottom:3px solid #99940f;color:#99940f;font-family:Arial;font-size:16px;letter-spacing:1px;text-align:center}.modal-content a{color:#99940f}.close{color:#24346e;float:right;font-size:28px;font-weight:700;margin:2px 8px 0 0}.close:focus,.close:hover{color:#99940f;cursor:pointer;text-decoration:none}.modal-header{background-color:#5cb85c;color:#fff}.modal-body,.modal-footer,.modal-header{padding:2px 16px}.modal-footer{background-color:#5cb85c;color:#fff}@-webkit-keyframes slideIn{0%{opacity:0;top:-300px}to{opacity:1;top:0}}@keyframes slideIn{0%{opacity:0;top:-300px}to{opacity:1;top:0}}@-webkit-keyframes fadeIn{0%{opacity:0}to{opacity:1}}@keyframes fadeIn{0%{opacity:0}to{opacity:1}}
```

Comment:
I have used some css minifier `cssnano` that is supposed to be smart and remove unnecessary. So looks like it is smaller than before but all looks fine for me, so maybe it works